boards/arm/contextualelectronics_abc/doc/index.rst

```diff
@@ -0,0 +1,149 @@
+
+* CLOCK
+* FLASH
+* :abbr:`GPIO (General Purpose Input Output)`
+* :abbr:`I2C (Inter-Integrated Circuit)`
+* :abbr:`MPU (Memory Protection Unit)`
+* :abbr:`NVIC (Nested Vectored Interrupt Controller)`
+* :abbr:`PWM (Pulse Width Modulation)`
+* Segger RTT (RTT Console)
+* :abbr:`SPI (Serial Peripheral Interface)`
+* :abbr:`UART (Universal asynchronous receiver-transmitter)`
+* Quectel BG95 Modem
+
+.. figure:: img/contextualelectronics_abc.jpg
+     :width: 2046px
+     :align: center
+     :alt: Contextual Electronics Advanced BLE Cell
+
+     Contextual Electronics Advanced BLE Cell (Credit: Chris Gamell)
+
+More information about the board can be found at the `ABC Board website`_.
+The `Nordic Semiconductor Infocenter`_ contains the processor's information
+and the datasheet.
+
+Hardware
+********
+
+ABC board has two external oscillators. The frequency of the slow clock
+is 32.768 kHz. The frequency of the main clock is 32 MHz.
+
+Supported Features
+==================
+
+The contextualelectronics_abc board configuration supports the following
+hardware features:
+
++-----------+------------+----------------------+
+| Interface | Controller | Driver/Component     |
++===========+============+======================+
+| CLOCK     | on-chip    | clock_control        |
++-----------+------------+----------------------+
+| FLASH     | on-chip    | flash                |
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| I2C(M)    | on-chip    | i2c                  |
++-----------+------------+----------------------+
+| MPU       | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| PWM       | on-chip    | pwm                  |
++-----------+------------+----------------------+
+| RTC       | on-chip    | system clock         |
++-----------+------------+----------------------+
+| RTT       | Segger     | console              |
++-----------+------------+----------------------+
+| SPI(M)    | on-chip    | spi                  |
++-----------+------------+----------------------+
+| UART      | on-chip    | serial               |
++-----------+------------+----------------------+
+| Modem     | on-board   | quectel_bg9x         |
++-----------+------------+----------------------+
+
+Other hardware features are not supported by the Zephyr kernel.
+See `ABC Board website`_ for more details on this board, and
+`Nordic Semiconductor Infocenter`_ for a complete list of SoC
+features.
+
+Programming and Debugging
+*************************
+
+Applications for the ``contextualelectronics_abc`` board configuration can be
+built and flashed in the usual way (see :ref:`build_an_application`
+and :ref:`application_run` for more details); however, the standard
+debugging targets are not currently available.
```

Comment:
Thanks for the update.

Taking a second look at this, that's not really true. The board.cmake file has all the necessary boilerplate to set up debugging.

I looked at the block diagram and I think the issue is just that you need an external debug probe. Perhaps just say that directly instead?

Reply:
Good catch ! This was a copy-paste from the nRF52840 DK doc. I've refactored it to look more similar to the adafruit_feather_nrf52840 readme.

Reply:
>  This was a copy-paste from the nRF52840 DK doc.

Ugh, which is also wrong ... :D. I'll fix that too, thanks for letting me know.